lib/workers/repository/update/pr/index.ts

```diff
@@ -97,16 +100,30 @@ export async function ensurePr(
   );
 
   const config: BranchConfig = { ...prConfig };
-
+  const filteredPrConfig = generatePrFingerprintConfig(config);
+  const prFingerprint = fingerprint(filteredPrConfig);
   logger.trace({ config }, 'ensurePr');
   // If there is a group, it will use the config of the first upgrade in the array
   const { branchName, ignoreTests, prTitle = '', upgrades } = config;
   const dependencyDashboardCheck =
     config.dependencyDashboardChecks?.[config.branchName];
-  // Check if existing PR exists
+  // Check if PR already exists
   const existingPr = await platform.getBranchPr(branchName);
+  const prCache = getPrCache(branchName);
   if (existingPr) {
     logger.debug('Found existing PR');
+    if (prCache) {
+      logger.trace({ prCache }, 'Found existing PR cache');
+      // return if pr cache is valid and pr was not changed in the past 24hrs
+      if (validatePrCache(prCache, prFingerprint)) {
+        logger.debug(
+          'PR cache matches and no PR changes in last 24hrs, so skipping PR body check'
```

Comment:
Why not move this debug log to `validatePrCache()` consistent with the other logging?